src/complex/universal.rs

```diff
@@ -23,6 +23,7 @@ pub const UNIVERSAL_SHAPER: ComplexShaper = ComplexShaper {
 };
 
 pub type Category = u8;
+#[allow(dead_code)]
```

Comment:
?

Reply:
It's because of this commit (already from a port in the past): https://github.com/harfbuzz/harfbuzz/commit/564afc425

It makes those 4 constants become unused, thus resulting in a warning:
```rs
    pub const G: u8 = 49; // HIEROGLYPH
    pub const J: u8 = 50; // HIEROGLYPH_JOINER
    pub const SB: u8 = 51; // HIEROGLYPH_SEGMENT_BEGIN
    pub const SE: u8 = 52; // HIEROGLYPH_SEGMENT_END
```

But since they haven't removed it from harfbuzz either, I just decided to surpress the warnings instead.